core/src/main/java/jenkins/model/Jenkins.java

```diff
@@ -5064,6 +5064,11 @@ private static void computeVersion(ServletContext context) {
 
     public static final PermissionGroup PERMISSIONS = Permission.HUDSON_PERMISSIONS;
     public static final Permission ADMINISTER = Permission.HUDSON_ADMINISTER;
+    /**
+     * Permission to read configuration.
+     * @see Item#EXTENDED_READ
```

Comment:
Huh?

Reply:
I'm afraid I don't understand what you are implying.